inc/iso22133object.hpp

```diff
@@ -76,11 +77,28 @@ class TestObject {
 	GeographicPositionType getOrigin() const { return origin; }
 	std::string getLocalIP() const { return localIP; }
 	uint32_t getTransmitterID() const { return transmitterID; }
+	uint32_t getServerID() const { return serverID; }
 	ObjectSettingsType getObjectSettings() const { return objectSettings; }
 
 
+	/** SWIG Wrappers **/
+	bool isAwaitingFirstHeab() { return awaitingFirstHeab;}
+	//! Set the endpoint for the process channel
+	void setProcessChannelEndpoint(int udpSocket, char *addr, const uint32_t port);
+	//! Wrapper for handling function that converts to char vector
+	int handleMessage(char *buffer, int bufferLen);
+	//! Wrapper for state change requests
+	void requestStateChange(ISO22133::Events::EventType event) { state->handleEvent(*this, event); }
```

Comment:
If the purpose of this is to change the object state I don't like it. The whole purpose of this is to not mess with the internal states.
If the purpose is something else, change the name to something more descriptive

Reply:
agreed

Reply:
problem with SWIG is that it makes it impossible to inherit from this object and use it in any meaningful way.